docs/book/getting-started/overview.md

```diff
@@ -4,7 +4,7 @@ This tutorial is intended to give an introduction to using Laminas by creating a
 
 ## Some assumptions
```

Comment:
The entire section "Some assumptions" should be removed and the system requirements should be added to the page "[Getting started: A skeleton application](https://docs.laminas.dev/tutorials/getting-started/skeleton-application/)".